packages/trackjs/package.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "trackjs",
```

Comment:
Can we do a "test" first? If you set name to "test-trackjs" we can test that publishing to eik etc works before we use the real name. This just to make sure we're not messing up a version on the real package name before its working. Deleting packages is a bit cumbersome on the CDN so if we can avoid that it would be super.

Reply:
sure